inc/storefront-template-functions.php

```diff
@@ -165,7 +165,7 @@ function storefront_site_branding() {
 			jetpack_the_site_logo();
 		} else { ?>
 			<div class="site-branding">
-				<h1 class="site-title"><a href="<?php echo esc_url( home_url( '/' ) ); ?>" rel="home"><?php bloginfo( 'name' ); ?></a></h1>
+				<div class="beta site-title"><a href="<?php echo esc_url( home_url( '/' ) ); ?>" rel="home"><?php bloginfo( 'name' ); ?></a></div>
```

Comment:
An interesting topic on MOZ: https://moz.com/community/q/h1-tag-as-logo-on-the-homepage